tests/nlu/data_augmentation/test_augmenter.py

```diff
@@ -0,0 +1,618 @@
+import operator
+import os
+from typing import Callable, List, Set, Text
+
+import pytest
+from rasa.nlu.constants import TOKENS_NAMES
+from rasa.nlu.tokenizers.tokenizer import Tokenizer
+from rasa.constants import DEFAULT_RANDOM_SEED
+from rasa.shared.nlu.training_data.message import Message
+import rasa.nlu.config
+import rasa.shared.nlu.training_data.loading
+import rasa.shared.utils.components
+import rasa.shared.utils.io
+from rasa.shared.nlu.constants import INTENT, TEXT, VOCABULARY, INTENT_REPORT_FILE_NAME
+
+
+def test_augmenter_create_tokenizer():
+    tokenizer_config = rasa.nlu.config.load("data/test_nlu_paraphrasing/config.yml")
+
+    tokenizer = rasa.shared.utils.components.get_tokenizer_from_nlu_config(
+        tokenizer_config
+    )
+
+    assert isinstance(tokenizer, Tokenizer)
+
+    # Test Config has a simple WhitespaceTokenizer
+    expected_tokens = ["xxx", "yyy", "zzz"]
+    message = Message(data={TEXT: "xxx yyy zzz", INTENT: "abc"})
+
+    tokenizer.process(message)
+    tokens = [token.text for token in message.get(TOKENS_NAMES[TEXT])]
+
+    assert tokens == expected_tokens
+
+
+@pytest.mark.parametrize(
+    "intent_proportion, comparator_fn_1, comparator_fn_2",
+    [
+        (0.0, operator.le, operator.lt),
+        (0.5, operator.ge, operator.le),
+        (1.0, operator.ge, operator.le),
+    ],
+)
+def test_augmenter_intent_collection(
+    intent_proportion: float, comparator_fn_1: Callable, comparator_fn_2: Callable
+):
+    from rasa.nlu.data_augmentation.augmenter import (
```

Comment:
why are we using a local import? Can we please refer to functions by their module and importing them directly?

Reply:
same for the other imports in this file